docs/crio.8.md

```diff
@@ -76,11 +78,23 @@ crio [GLOBAL OPTIONS] config [OPTIONS]
 
 **--cpu-profile**="": set the CPU profile file path
 
-**--default-transport**: A prefix to prepend to image names that can't be pulled as-is.
+**--default-capabilities**="": capabilities to add to the containers
```

Comment:
Could you list the default-capabilities here.

Reply:
Fixed